### PR TITLE
L1-3476: Added the password reset form and endpoint to the SDK

### PR DESCRIPTION
**Pull request by babadee08**
https://app.clubhouse.io/venture-leap/story/3476/l1-finalize-leap-one-demo-app-by-using-pw-reset-and-registration-mail-endpoints

### CODE REVIEW

File: src/Form/Type/UserPasswordRequestType.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use VentureLeap\LeapOnePhpSdk\Model\User\User;
+
+class UserPasswordRequestType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('email', TextType::class)
```
Comment (jazzzzy) [inline src/Form/Type/UserPasswordRequestType.php:20]: EmailType::class could be used here

Comment (jazzzzy) [reply src/Form/Type/UserPasswordRequestType.php:20]: Does it also validates the email?

**Review by jazzzzy — APPROVED** [review]
👍 Just some minor comments. Please take a look

File: src/Resources/views/User/login.html.twig [hunk 2]
```diff
@@ -15,6 +15,12 @@
 
         {{ form(form) }}
 
+        <div>
+            <span>
+                <a href="{{ path('leap_one_user_reset_password') }}">Forgot password?</a>
```
Comment (escalantef) [inline src/Resources/views/User/login.html.twig:20]: Maybe translation

File: src/Resources/views/User/passwordRequest.html.twig [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html.twig' %}
+
+{% block content %}
+    <h1>Please enter your email</h1>
```
Comment (escalantef) [inline src/Resources/views/User/passwordRequest.html.twig:4]: Also here

**Review by escalantef — APPROVED** [review]
Maybe we could add translations. I know that I didn't do it at the begging :smile:

**Pull request merged**